src/ValidatorKeys.cpp

```diff
@@ -0,0 +1,182 @@
+//------------------------------------------------------------------------------
+/*
+    This file is part of validator-keys-tool:
+        https://github.com/ripple/validator-keys-tool
+    Copyright (c) 2016 Ripple Labs Inc.
+
+    Permission to use, copy, modify, and/or distribute this software for any
+    purpose  with  or without fee is hereby granted, provided that the above
+    copyright notice and this permission notice appear in all copies.
+
+    THE  SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+    WITH  REGARD  TO  THIS  SOFTWARE  INCLUDING  ALL  IMPLIED  WARRANTIES  OF
+    MERCHANTABILITY  AND  FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+    ANY  SPECIAL ,  DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+    WHATSOEVER  RESULTING  FROM  LOSS  OF USE, DATA OR PROFITS, WHETHER IN AN
+    ACTION  OF  CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+    OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+*/
+//==============================================================================
+
+#include <beast/core/detail/base64.hpp>
+#include <json_reader.h>
+#include <ripple/protocol/HashPrefix.h>
+#include <ripple/protocol/Sign.h>
+#include <ValidatorKeys.h>
```

Comment:
Although it goes against the general guideline of listing include files alphabetically, I'll suggest that you put `<ValidatorKeys.h>` first in the list.  Listing the header that corresponds to this `.cpp` file first helps to ensure that the header can stand alone and does not have dependencies on other headers that it does not manage for itself.  This is a guideline from "Large Scale C++ Software Design" Section 3.2, Physical Design Rules.